packages/renderer/src/lib/editor/MonacoEditor.svelte

```diff
@@ -54,6 +54,10 @@ onMount(async () => {
     editor.dispose();
   };
 });
+
+export function setContent(content: string) {
```

Comment:
I would expect that if exported content variable change, then it's updated. No need to call a setContent method